docs/farming-&-staking/wallets/polkadot.md

```diff
@@ -1,7 +1,7 @@
 ---
 title: Polkadot.js
 sidebar_position: 2
-description: How to configure the Polkadot Substrate wallet for the Subspace Network
+description: How to configure the Polkadot Substrate wallet for the Autonomys Network
```

Comment:
Let's please keep this as "Subspace" as the name Autonomys is not yet supported by PolkadotJS